.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media
```

Comment:
There is an extra space before the `media` directory. It should be removed to ensure the directory is properly ignored.